src/test/lrsql/admin/cors_test.clj

```diff
@@ -72,7 +72,7 @@
         (is-err-code (create-account bad-cors-headers req-body) 403)))
     (testing "create account with default CORS check success"
       (let [good-cors-headers
-            (merge headers {"Origin" "http://0.0.0.0:8080"})
+            (merge headers {"Origin" "http://localhost:8080"})
```

Comment:
I'm assuming that you're overriding the existing tests, as opposing to adding new tests, since all places where `0.0.0.0` works should also work with `localhost`, right?

Reply:
yep! Figured localhost is just better from an instructive standpoint